src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/set_interior_vehicle_data_request_test.cc

```diff
@@ -212,12 +217,13 @@ TEST_F(
       .WillOnce(Return(true));
 
   // Act
-  std::shared_ptr<rc_rpc_plugin::commands::SetInteriorVehicleDataRequest>
-      command = CreateRCCommand<
-          rc_rpc_plugin::commands::SetInteriorVehicleDataRequest>(
-          mobile_message);
+  auto command = CreateRCCommand<SetInteriorVehicleDataRequest>(mobile_message);
   ASSERT_TRUE(command->Init());
   command->Run();
+
+  EXPECT_FALSE(control_data.keyExists(kState));
+  EXPECT_TRUE(control_data.keyExists(kRadioEnable) &&
```

Comment:
@mvorobio please transform this expression to 2 separate expectations.

Reply:
@AByzhynar please see above.